next/backend/utils/fetchers/blogPostsFetcher.ts

```diff
@@ -0,0 +1,65 @@
+// @ts-strict-ignore
```

Comment:
I understand this approach, but I removed them in first commit and just put them back, so no more ignores were added :)
I don't completely understand in what stage we are, because strict mode is turned off, as far as I see in tsconfig.